pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -173,32 +173,20 @@ func (p *DefaultProvider) createAMIOptions(ctx context.Context, nodeClass *v1bet
 	if len(nodeClass.Status.SecurityGroups) == 0 {
 		return nil, fmt.Errorf("no security groups are present in the status")
```

Comment:
How should we track docs changes with these kinds of things?

Reply:
I couldn't find any explicit documentation for this behavior, so there are no updates required to existing docs. I'm not sure what we want to do for the upgrade guide, I could put in a placeholder not but I think we can also just rely on tracking this in the v1 roadmap and construct the upgrade guide from there.